<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSH</class>
 <widget class="QWidget" name="LSH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSH</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="tune_btn">
     <property name="text">
      <string>Tune</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="group_gb">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Current LSH grid:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>LSH grids:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="7">
       <widget class="QTableWidget" name="lshGrids_tw"/>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="add_btn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="apply_btn">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="7">
       <widget class="QLineEdit" name="file_le"/>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QPushButton" name="remove_btn">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="6">
       <widget class="QCheckBox" name="selectAll_cb">
        <property name="text">
         <string>Select all</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QLabel" name="info_lbl">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
